### PR TITLE
update links in README.md

### PR DESCRIPTION
**Pull request by morrme**
Fixes: #5840 

Modified text in "lint tool" section and updated inaccurate links.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/5846)

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/229830261)

# Firefox (nightly channel) #

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/229830267)

# Chrome (unstable channel) #

File: README.md [hunk 1]
```diff
@@ -231,13 +231,14 @@ request, and reviewers will not merge branches with tests that have
 lint errors, so you must fix any errors the lint tool reports. For
 details on doing that, see the [lint-tool documentation][lint-tool].
 
-But in the unusual case of error reports for things essential to a
+In the unusual case of error reports for things essential to a
 certain test or that for other exceptional reasons shouldn't prevent
 a merge of a test, update and commit the `lint.whitelist` file in the
-web-platform-tests root directory to suppress the error reports. For
-details on doing that, see the [lint-tool documentation][lint-tool].
+web-platform-tests root directory to suppress the error reports.
```
Comment (xfq) [inline README.md]: Nit: trailing whitespace.

File: README.md [hunk 2]
```diff
@@ -231,13 +231,14 @@ request, and reviewers will not merge branches with tests that have
...
 certain test or that for other exceptional reasons shouldn't prevent
 a merge of a test, update and commit the `lint.whitelist` file in the
-web-platform-tests root directory to suppress the error reports. For
-details on doing that, see the [lint-tool documentation][lint-tool].
+web-platform-tests root directory to suppress the error reports. 
+
+For more details, see the [lint-tool documentation][lint-tool].
 
-[lint-tool]: https://github.com/w3c/web-platform-tests/blob/master/docs/lint-tool.md
+[lint-tool]: /docs/_writing-tests/lint-tool.md
```
Comment (xfq) [inline README.md]: http://web-platform-tests.org/writing-tests/lint-tool.html might be better, I think.

**Comment by jgraham** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/w3c/web-platform-tests/5846)*

**Pull request merged**